cmd/root.go

```diff
@@ -17,6 +18,16 @@ var rootCmd = &cobra.Command{
 	Args:    cobra.MinimumNArgs(1),
 }
 
+func init() {
+	cobra.OnInitialize(initConfig)
```

Comment:
Do we know if calling this multiple times is bad? Or necessary? This is being called in check.go as well. It seems that we would only want this once (especially since this is in an init() func).

Reply:
It didn't hurt but it's unnecessary so I removed the call from check.go leaving the root.go intact.

Reply:
I think there are two patters to call `OnInitialize` 
- in the `init()` in the `root.cmd`
- in the `main`
and in both instances it's only called once. So we probably only want to call this once where it already is.

Reply:
@acornett21 you mean to keep the one in root.go or in check.go

Reply:
Cool. That's where I left it.